docs/en/table_design/StarRocks_table_design.md

```diff
@@ -2,74 +2,134 @@
+| uid         | int         | YES  | true  | NULL    |       |
+| name        | varchar(64) | YES  | true  | NULL    |       |
+| age         | int         | YES  | false | NULL    |       |
+| phone       | varchar(16) | YES  | false | NULL    |       |
+| last_access | datetime    | YES  | false | NULL    |       |
+| credits     | double      | YES  | false | NULL    |       |
++-------------+-------------+------+-------+---------+-------+
+6 rows in set (0.00 sec)
+```
 
-In summary, StarRocks performs the following five steps to locate a row of a table by using the prefix that is composed of the dimension columns for the row:
+Execute [SHOW CREATE TABLE](../sql-reference/sql-statements/data-manipulation/SHOW_CREATE_TABLE.md) to view the CREATE TABLE statement.
 
-1. Search the prefix index table to locate the sequence number of the starting row in the data block taken by the row of interest.
+```SQL
+MySQL [example_db]> SHOW CREATE TABLE user_access\G
+*************************** 1. row ***************************
+       Table: user_access
+Create Table: CREATE TABLE `user_access` (
+  `uid` int(11) NULL COMMENT "",
+  `name` varchar(64) NULL COMMENT "",
+  `age` int(11) NULL COMMENT "",
+  `phone` varchar(16) NULL COMMENT "",
+  `last_access` datetime NULL COMMENT "",
+  `credits` double NULL COMMENT ""
+) ENGINE=OLAP 
+DUPLICATE KEY(`uid`, `name`)
+DISTRIBUTED BY RANDOM
+PROPERTIES (
+"replication_num" = "3",
+"in_memory" = "false",
+"enable_persistent_index" = "false",
+"replicated_storage" = "true",
+"compression" = "LZ4"
+);
+1 row in set (0.00 sec)
+```
 
-2. Search the row number index table of each dimension column to locate the data blocks for the dimension column.
+## Understand comprehensive table structure
 
-3. Read the data blocks.
+A deep dive into StarRocks table structures helps you design efficient data management structure tailored  to your business needs.
 
-4. Decompress and decode the data blocks.
+### [Table types](./table_types/table_types.md)
 
-5. Search the data blocks to locate the row onto which the dimension column index is mapped.
+StarRocks provides four types of tables which are Duplicate Key tables, Primary Key tables, Aggregate tables, and Unique Key tables, to store data for various business scenarios, such as raw data, frequently updated realtime data, and aggregated data.
 
-## Accelerated processing
+- Duplicate Key tables are simple and easy to use. No constraint is added to columns in this type of table, so duplicate data rows exist in the table. Duplicate Key tables are suitable for storing raw data, such as logs, that does not need any constraints or pre-aggregation.
+- Primary Key tables are powerful. Both unique and non-null constraints are added to primary key columns. Primary Key tables support real-time frequent updates and partial column updates, while ensuring high query performance, and therefore are suitable for real-time queries scenarios.
```

Comment:
Both unique and non-null constraints are added to primary key columns. --- not easy to understand. The primary key columns must have unique and NOT NULL values.